packages/wrapped-keys/project.json

```diff
@@ -11,7 +11,14 @@
         "outputPath": "dist/packages/wrapped-keys",
         "main": "packages/wrapped-keys/src/index.ts",
         "tsConfig": "packages/wrapped-keys/tsconfig.lib.json",
-        "assets": ["packages/wrapped-keys/*.md"],
+        "assets": [
+          "packages/wrapped-keys/*.md",
+          {
+            "glob": "**/*",
+            "input": "packages/wrapped-keys/src/lib/generated/litActions",
+            "output": "src/lib/generated/litActions"
+          }
```

Comment:
Why is this explicitly required to be defined as all these files are already `.js`?

Reply:
Is the idea that we run the bundler script separately for the `@lit-protocol/wrapped-keys-bc` release? And for `@lit-protocol/wrapped-keys` these files don't exist so they won't be included in the final build hence it's required to explicitly define it here?